### ISSUE TITLE
make test_prep: new error output

### ISSUE THREAD

**Issue by jkeenan** [Perl/perl5#19779]
Running `make test_prep` at the following commit and configuration ...
```
$ git describe
v5.37.0-49-ge1980ad
$ ./perl -Ilib -V:config_args
config_args='-des -Dusedevel -Duseithreads';
```
... I now get the following messages near the very end of the process:
```
./miniperl -Ilib autodoc.pl
embed.fnc entry overrides redundant information in 'Am|bool|SvPVXtrue|SV * sv' in inline.h at autodoc.pl line 519, <F> line 321.
no docs for do_close
```

**Comment by jkeenan** [comment]
> Running `make test_prep` at the following commit and configuration ...
> 
> ```
> $ git describe
> v5.37.0-49-ge1980ad
> $ ./perl -Ilib -V:config_args
> config_args='-des -Dusedevel -Duseithreads';
> ```
> 
> ... I now get the following messages near the very end of the process:
> 
> ```
> ./miniperl -Ilib autodoc.pl
> embed.fnc entry overrides redundant information in 'Am|bool|SvPVXtrue|SV * sv' in inline.h at autodoc.pl line 519, <F> line 321.
> no docs for do_close
> ```

I did not observe these when building perl-5.36.0, so I believe these have been introduced since 5.37.0 came out last night.

**Comment by jkeenan** [comment]
Possible source:  this commit merged to blead this morning:
```
commit dec3ff31f81782e2cb89a00e8be63635ae97430a
Author:     Karl Williamson <khw@cpan.org>
AuthorDate: Mon Apr 25 06:01:39 2022 -0600
Commit:     Karl Williamson <khw@cpan.org>
CommitDate: Sat May 28 08:52:16 2022 -0600

    Evaluate SvPVXtrue's argument just once
```
@khwilliamson, can you take a look?

**Comment by khwilliamson** [comment]
Fixed by e1fa7ce9e264f21f1bbb000feaf336040252cd64

**Comment by jkeenan** [comment]
> Fixed by [e1fa7ce](https://github.com/Perl/perl5/commit/e1fa7ce9e264f21f1bbb000feaf336040252cd64)

Confirmed at bdadb244dbc743bdb32c9b5f33d0940d55b93f8d.  Thanks.
